pkg/utils/sysctl.go

```diff
@@ -0,0 +1,24 @@
+package utils
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strconv"
+
+	"github.com/golang/glog"
+)
+
+func SetSysctl(path string, value int) error {
+
+	sysctlPath := fmt.Sprintf("/proc/sys/%s", path)
+	if _, err := os.Stat(sysctlPath); err != nil {
+		if os.IsNotExist(err) {
+			glog.Infof("%s not found, could not set value: %s, Does your kernel support this feature?", sysctlPath)
```

Comment:
should we be returning err here instead of just logging?

Reply:
I feel like the caller of this method should be logging if anything. Maybe a new error type is needed if we don't want to "exit" on error?

Reply:
I don't understand what you are suggesting. could you provide some example of how this new error type would work?

Reply:
i got it and have done what you asked for :)